metaflow/cli.py

```diff
@@ -1013,11 +1021,9 @@ def start(
     if ctx.invoked_subcommand not in ("run", "resume"):
         # run/resume are special cases because they can add more decorators with --with,
         # so they have to take care of themselves.
-        all_decospecs = ctx.obj.tl_decospecs + list(
-            ctx.obj.environment.decospecs() or []
-        )
-        if all_decospecs:
-            decorators._attach_decorators(ctx.obj.flow, all_decospecs)
+        decospecs = list(decospecs or []) + list(ctx.obj.environment.decospecs() or [])
```

Comment:
nit, use ctx.obj.decospecs (no point redoing the same computation and conversion)